ddtrace/contrib/multiprocessing/patch.py

```diff
@@ -0,0 +1,104 @@
+from ddtrace.vendor.wrapt import wrap_function_wrapper as _w
+
+import multiprocessing
+
+try:
+    # py3
+    from multiprocessing.process import BaseProcess as Process
+except ImportError:
+    from multiprocessing.process import Process
+from multiprocessing.pool import Pool
+
+from ... import Pin, config
+from ...ext import SpanTypes
+from ...internal.logger import get_logger
+from ...utils.formats import asbool, get_env
+from ...utils.importlib import func_name
+from ...utils.wrappers import unwrap as _u
+
+log = get_logger(__name__)
+
+DATADOG_PATCHED = "__datadog_patch"
+DATADOG_PIN = "__dd_pin"
+DATADOG_CONTEXT = "__datadog_context"
+
+# Configure default configuration
+config._add(
+    "multiprocessing",
+    dict(
+        service_name=get_env("multiprocessing", "service_name", "multiprocessing"),
+        app="multiprocessing",
+        distributed_tracing=asbool(get_env("multiprocessing", "distributed_tracing", True)),
```

Comment:
This isn't used, and doesn't necessarily make sense as a setting either (I think).